tests/unit/mongodb_test_charm/charmcraft.yaml

```diff
@@ -30,3 +30,5 @@ parts:
     source: .
     prime:
       - workload_version
+      - charm_version
+      - charm_internal_version
```

Comment:
note we will be able to remove this in the future

Reply:
Yes and I'll start first with the internal version following your excellent work on the current charms :)